tools/clang/unittests/HLSLExec/ShaderOpTest.h

```diff
@@ -31,7 +31,8 @@
 ///////////////////////////////////////////////////////////////////////////////
 // Forward declarations.
 namespace dxc {
-class DxcDllSupport;
+class DllLoader;
+class SpecificDllLoader;
```

Comment:
Do we need both of these?

Reply:
Yes, without them the uses in 364 / 369 fail to compile.